worlds/tunic/er_scripts.py

```diff
@@ -381,7 +463,7 @@ def create_randomized_entrances(portal_pairs: Dict[Portal, Portal], regions: Dic
         region2 = regions[portal2.region]
         region1.connect(region2, f"{portal1.name} -> {portal2.name}")
         # prevent the logic from thinking you can get to any shop-connected region from the shop
-        if portal2.name != "Shop":
+        if not portal2.name.startswith("Shop"):
```

Comment:
Same question

Reply:
This specific bit right here is to stop you from having a shop -> non-shop connection, since as you know, the shop only brings you back to wherever you just were, rather than any of the shop entrances.